docs/_posts/examples/3400-01-26-game-controls.html

```diff
@@ -0,0 +1,133 @@
+---
+layout: example
+category: example
+title: Game-like controls
```

Comment:
The example title should be a verb phrase

 > Navigate the map with game-like controls